new-test-case.mdx

```diff
@@ -1,44 +1,70 @@
 ---
 title: Create a new test case
-description: "An overview of the various test case creation methods"
+description: "Add more test cases to your test suite. Use AI generation or recording."
 icon: "arrow-progress"
 ---
 
-## Prompting for test case generation
+## Prompt our AI agent to generate new tests
 
-We are deploying an `AI agent` to find the interactions leading towards a desired user flow.
-While our `AI agent` finds the steps of your test cases, it informs you about the progress it's making by showing the steps.
+The fastest way to create a new test case is to have it AI generated. You give us a short prompt and we deploy our `AI agent` to find the interactions leading towards a desired user flow.
 
-<Frame caption="AI agent informs about its progress, screenshot 02/2024">
+<Frame caption="Create new test case by prompting the AI agent, screenshot 07/2024">
   <img
-    src="images/AI-Agent-generating-steps.png"
+    src="images/prompting/create-new-test.png"
     alt="AI agent informs about its progress"
   />
 </Frame>
 
-### Editing your AI discovery
+While our `AI agent` generates the steps of your test cases, it informs you about its progress.
 
-If the Agent generated wrong steps, you can modify the prompt and decide which steps to keep. All subsequent steps are replaced with the new agent output.
+<Frame caption="AI agent informs about its progress, screenshot 07/2024">
+  <img
+    src="images/prompting/AI-Agent-generating-steps.png"
+    alt="AI agent informs about its progress as it generates steps"
+  />
+</Frame>
+
+If the AI agent generated wrong steps or signals a failed step with a yellow alert, you can:
 
-<Frame caption="Relaunch AI discovery based on your comments, screenshot 02/2024">
+1. **Restart the AI generation process.** Select the last step you want to keep. All subsequent steps will be replaced with the new AI agent output.
+2. Try a **different prompt** and restart the AI generation process.
+3. **Add, edit and remove steps manually.** Our virtual locator picker helps you edit tests in no time.
+
+<Frame caption="A generated test step failed, hightlighted by yellow alert, screenshot 07/2024">
   <img
-    src="images/AI-Agent-recreate-steps.png"
-    alt="Relaunch AI discovery based on comments"
+    src="images/prompting/failed-test-step.png"
+    alt="A generated step failed and shows a yellow alert"
   />
 </Frame>
 
-### Chaining your tests
+### Pre-prompted login test
+
+If your app has a login functionality that wasn't AI discovered at the set-up, we have a pre-prompted login test case for you. You can trigger it after giving us test user credentials.
+
+Go to `settings` a.k.a. the `gear icon` in the upper right corner and select `test account`. Add test user credentials we can use to run the login test.
```

Comment:
Do we want to mention that this is also created on the fly for a new project, if we detect a login test case and the user provides credentials there?

Reply:
aaaah, good one. I'll do a new commit.